app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Great job working on the set up with devise but I think it would be great if you add :confirmable as well here since in the requirements mentioned that you should Ask for confirmation of email. so kindly add it and after siging up the user should receive a link to ask for confirmation  and I think for now you could see the link in the terminal not in a real email adress so please add it 💯🚀😄🔥


![image](https://github.com/OsvaldoBC/blog/assets/58553711/e7ffb033-0f23-4f52-a5cb-7aa95bd84c84)